Week2/index6.js

```diff
@@ -0,0 +1,16 @@
+//6
+let myfavouriteanimals = [];
+
+console.log("My favourite animals");
+
+//console.log(myfavouriteaninals); --> not defined
```

Comment:
`myfavouriteaninals` is not defined because you misspelled it;) you wrote ani**n**als instead of ani**m**als
If you do:
`console.log(myfavouriteanimals);`
then it will work :)